tests/stress_test.sh

```diff
@@ -302,34 +360,36 @@ if [[ "${failed}" -gt 0 ]]; then
   # Showing failed tests and removing temporary file
   printf "\nFailed tests:\n" | tee -a "${results_file}" | color $magenta
 
-  while IFS= read -r line; do
-    if [[ $(echo $line | wc -w) -eq 1 ]]; then
-      printf "$line\n" | tee -a "${results_file}" | color $red
-    elif [[ $line == " - "* ]]; then
-      printf "$line\n" | tee -a "${results_file}" | color $yellow
-    else
-      printf "$line\n" | tee -a "${results_file}"
-    fi
-  done <failed_tmp.txt
-  rm failed_tmp.txt
+  find . -type f -name "failed_tmp.txt" | while IFS= read -r file; do
+    while IFS= read -r line; do
+      if [[ $(echo $line | wc -w) -eq 1 ]]; then
+        printf "$line\n" | tee -a "${results_file}" | color $red
+      elif [[ $line == " - "* ]]; then
+        printf "$line\n" | tee -a "${results_file}" | color $yellow
+      else
+        printf "$line\n" | tee -a "${results_file}"
+      fi
+    done < "$file"
+    rm "$file"
+  done
   printf "\n" | tee -a "${results_file}"
 fi
 
-# Removing temporary files
-rm ${tmp_file}
-
 printf "Results saved to ${results_dir}\n\n"
 
 # Prompting for file deletion
-printf "Clear ALL *.geojson, *.png and *.ps files in directory? [y/N]: " | color $yellow
+printf "Clear ALL *.geojson, *.csv, *.svg, *.png and *.ps files in directory? [y/N]: " | color $yellow
 read -t 10 to_clear
 if [[ $? -ne 0 ]]; then
   printf "\nNo response received! " | color $red
 elif [ "$to_clear" == "y" ]; then
-  rm *.geojson
-  rm *.png
-  rm *.ps
-  printf "All *.geojson, *.png and *.ps files deleted.\n" | color $red
+  # Clear the files in subdirectories as well if flags are enabled
+  find . -name "*.geojson" -type f -delete
+  find . -name "*.csv" -type f -delete
+  find . -name "*.svg" -type f -delete
+  find . -name "*.png" -type f -delete
+  find . -name "*.ps" -type f -delete
+  printf "All *.geojson, *.csv, *.svg, *.png and *.ps files deleted.\n" | color $red
   exit ${failed}
 fi
 
```

Comment:
At the end, it would be useful to display summary statistics grouped by flags. For example: 5 failures without any flag, 3 failures with the `plot_polygons` flag, etc. Additionally, for each failed test, we should display the exact command that can be run from the root folder to reproduce it.